src/openedx_companion_auth/apps.py

```diff
@@ -0,0 +1,19 @@
+"""openedx_companion_auth app config"""
+
+from django.apps import AppConfig
+
+
+class MITxCoreConfig(AppConfig):
+    """App configuration"""
+
+    name = "openedx_companion_auth"
```

Comment:
Sorry, I thought I had commented on this earlier. I think a more suitable name w.r.t the other plugins in this mono repo would be `ol_openedx_companion_auth`.

@blarghmatey do you have any reservations about this?